public_html/i18n/fr.json

```diff
@@ -0,0 +1,6 @@
+{
+	"name": "CopyPatrol",
+	"wikiprojects": "WikiProjets",
```

Comment:
Yup! It's French: https://fr.wikipedia.org/wiki/Aide:Projet

But it's a good point: I didn't translate 'CopyPatrol'... is it "CopiePatrouille"? Or "patrouille du droit d'auteur" as google says?... :-) We need a French speaker here...